ops/terraform/aws/deployment/asg/main.tf

```diff
@@ -0,0 +1,146 @@
+provider "aws" {
+  region = "${var.region}"
+}
+
+# Application VPC and subnet lookup
```

Comment:
We might want to use comments here to indicate that the templated naming/variable substitution here must be kept in sync with that of the networking TF config file.